src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -268,6 +268,17 @@ extern void setDbPrefix(std::string _dbPrefix)
 
 /*****************************************************************************
 *
+* dbPrefixGet -
+*/
+const char* dbPrefixGet(void)
```

Comment:
In my opinion this is clearly a case in which readibility is most important that performance. In addition, std::string has been in CB from the very beginning. Thus, I would suggest to continue with the same style.